NuKeeper/Commands/GitHubNuKeeperCommand.cs

```diff
@@ -1,136 +1,142 @@
-using System;

```

Comment:
There's EOL changes here - is that a mismatch in your settings or is the file in GitHub already mangled? If that the latter, I'd rather do a EOL fix commit separately first.

Reply:
I think that's on your side - added #423 which should prevent line endings issues in the future (unless, of course, someone uses a single checkout both with Linux and Windows tooling...)

Reply:
It appears that #420 swapped from LF to CRLF and as we merged conflicts we unknowingly switched it back to LF. I've added a comment in #423 as such.